CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changes
+
+## 0.4.0
+* [Changed] Set configured tags on plugin generated logs. (See [#45](https://github.com/DataDog/jmeter-datadog-backend-listener/pull/45)).
```

Comment:
I set this to `Changed` since it was already released manually as `Changed`: https://github.com/DataDog/jmeter-datadog-backend-listener/releases/tag/0.4.0, although I wonder if `Added` might be better, since `Changed` should be for breaking changes?

Reply:
Technically this is changing behavior (even if it's additive in that it adds more tags), that was my thinking behind it.